client/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "client",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^16.10.1",
```

Comment:
is it possible that this doesn't include [webpack](https://webpack.js.org)?

Reply:
Yes, that's normal!